scripts/python/archive_repositories.py

```diff
@@ -60,33 +50,51 @@ def ready_for_archiving(repository, archive_date) -> bool:
     return False
 
 
-# How long ago in which the repositories should be archived
-archive_date_days = 0
-archive_date_months = 6
-archive_date_years = 1
-
-archive_date = datetime.now() - relativedelta(
-    days=archive_date_days, months=archive_date_months, years=archive_date_years
-)
-
-# Create MoJGithub object
-moj_gh = MojGithub(org=organization, org_token=org_token)
-
-# Get all repos that need archiving
-repos = [
-    repo
-    for repo in moj_gh.get_unarchived_repos("public")
-    if ready_for_archiving(repo, archive_date)
-]
-
-# Print repos
-logging.info(
-    f"Beginning archive of inactive repositories for GitHub organization: {organization}"
-)
-logging.info("-----------------------------")
-logging.info(f"Searching for inactive repositories from date: {archive_date}")
-logging.info("-----------------------------")
-
-# Archive repos
-for repo in repos:
-    MojArchive(repo).archive()
+def main():
+    # Config
+    # Logging Config
+    logging.basicConfig(
+        format="%(asctime)s %(levelname)-8s %(message)s",
+        level=logging.INFO,
+        datefmt="%Y-%m-%d %H:%M:%S",
+    )
+
+    # Change this to point at a different GitHub organization
+    organization = "ministryofjustice"
+    org_token = os.getenv("ADMIN_GITHUB_TOKEN")
+
+    # How long ago in which the repositories should be archived
+    archive_date_days = 0
+    archive_date_months = 6
+    archive_date_years = 1
+
+    archive_date = datetime.now() - relativedelta(
+        days=archive_date_days, months=archive_date_months, years=archive_date_years
+    )
+
+    # Create MoJGithub object
+    moj_gh = MojGithub(org=organization, org_token=org_token)
+
+    # Get all repos that need archiving
+    repos = [
+        repo
+        for repo in moj_gh.get_unarchived_repos("public")
```

Comment:
kept this as `public` - although the analytical services version has it as `all`. Not sure if we have a preference?

Reply:
I'm pretty sure we do all? @NickWalt01

Reply:
The AP Org, are mostly internal and private, with few public repos. It is the opposite on MoJ Org, most are public. We could go with all across both. @AntonyBishop any issues with that?

Reply:
Merged in, one to think about when we start deleting the MOJAS scripts and combining the two, I think 😁